package.json

```diff
@@ -57,6 +58,7 @@
     "sinon": "~2.1.0",
     "socket.io": "~1.7.3",
     "socket.io-client": "~1.7.3",
+    "sweetalert": "^1.1.3",
```

Comment:
I'm not sure how your sweetalert worked. And how it got to the `public/lib` folder. This should be in `bower.json` or maybe just get it as a CDN https://cdnjs.com/libraries/sweetalert